ktxspec.adoc

```diff
@@ -1930,6 +1981,39 @@ seconds is stem:[duration / timescale].
 This metadata entry must not be used together with
 <<KTXcubemapIncomplete,`KTXcubemapIncomplete`>>.
 
+=== KTXmapRange
+When the payload is a floating point format with a restricted range,
+such as ASTC HDR, RGB9E5 and FLOAT16, and data is input from a
+source with 32-bit floating point data, such as a .exr file,
+the values in the source may be mapped to the more limited range.
+KTX file writers may indicate the mapping by using the key
+
+- `KTXmapRange`
+
+The value is 8 bytes representing two IEEE 754 single-precision
+floating-point values:
+[source,c]
+----
+Float32 scale
+Float32 offset
+----
+
```

Comment:
Maybe mention here that these two values should be finite.

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Specification/pull/216/commits/b4fa36e9730b8591d135ec5beef44079d1521d3f.